### ISSUE TITLE
Moving the mouse toward the top left corner of the app causes Table Alignment hover menu to shows up.

### ISSUE THREAD

**Issue by LunaciaDev** [Zettlr/Zettlr#5190]
### Description

When a note having a markdown file containing a table is opened and the Table Editor option enabled, hovering the mouse to the left of the "File manager" button (or to the left of "File button" with Native Window Appearance disabled) on the toolbar will cause the Table Alignment hover menu to appear when it should not.

### Reproducing

1. Open Preference > Editor and make sure Table Editor is enabled
2. Create an empty Markdown file
3. Create a Table

If Native Windows Appearance option is enabled, hovering the mouse to the left of "File manager" button on the toolbar will cause the Table Alignment hover menu to show up.

If Native Windows Appearance option is disabled, hovering the mouse to the left of "File" button on the toolbar will cause the Table Alignment hover menu to show up.

### Zettlr Version

Stable (most recent version)

### Specify version

3.1.1

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Fedora 39 (Xfce)

### Additional Information

It created many instances of all table hover items when the bug happen.

![image](https://github.com/Zettlr/Zettlr/assets/130038743/dc38c089-ccfc-481f-9f95-97c03e7b3368)

**Comment by nathanlesage** [comment]
TL;DR: Just disable the TableEditor.

That thing is horribly broken and I want to tackle that task over the summer.
